embeddings/evaluator/submission.py

```diff
@@ -0,0 +1,100 @@
+import json
+import tempfile
+import zipfile
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Type, Union
+
+import srsly
+import yaml
+from wandb.apis.public import Run
+
+from embeddings.data.io import T_path
+from embeddings.evaluator.evaluation_results import Predictions
+from embeddings.evaluator.sequence_labeling_evaluator import SequenceLabelingEvaluator
+from embeddings.evaluator.text_classification_evaluator import TextClassificationEvaluator
+from embeddings.utils.json_dict_persister import CustomJsonEncoder
+
+
+@dataclass
+class Submission:
+    submission_name: str
+    dataset_name: str
+    dataset_version: str
+    embedding_name: str
+    metrics: Dict[str, Any]
+    hparams: Dict[str, Any]
+    predictions: Predictions
+    packages: List[str]
+    config: Optional[Dict[str, Any]] = None  # any additional config
+
+    @staticmethod
+    def from_run(
```

Comment:
Could we also have method `from_local_files`, where there will be paths to files as parameters?

Reply:
yeah, but is it a priority right now?